### PR TITLE
Origin/viber templates

### PR DESCRIPTION
**Pull request by joshuajott**
Added Viber documentation.  
Includes Structured Content information, Capabilities, and examples

### CODE REVIEW

File: _data/capabilities.yaml [hunk 1]
```diff
@@ -13,6 +13,7 @@
     - messagingWindowAPI: "Supported"
     - connectorAPI: "Supported"
     - weChat: "N/A in Channel"
+    - viber: "N/A in Channel"
```
Comment (cbikar) [inline _data/capabilities.yaml:16]: @joshuajott are we using this data anywhere? I think capabilities list moved to KC

File: pages/Documents/RichMessaging/ViberTemplates/carousel-template.md [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+The Viber carousel is a horizontally scrollable carousel of up to 10 vertical rich cards. Each card can include the following elements:
+
+* Title text
+* Subtitle text
+* Image (which will be placed above the title, subtitle and buttons) - image is not required to be placed in a card
+* Buttons with the following actions:
+    * Link
+    * Publish text
+
+Note:  When building a Viber carousel card you must keep in mind that each card consists of a total of 7 rows. Each row is an element
```
Comment (cbikar) [inline pages/Documents/RichMessaging/ViberTemplates/carousel-template.md:24]: @joshuajott can we catch that on the connector and reject the message if there is a card with more than 7 rows?

File: pages/Documents/RichMessaging/ViberTemplates/limitations.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  sitesection: Documents
+  categoryname: "Rich Messaging"
+  documentname: Viber Messaging Templates
+  permalink: viber-templates-limitations.html
+  indicator: messaging
+---
+
+### General limitations
+
+* A rich card can contain any or all of the supported elements: image, title, subtitle, buttons. However, it **must contain at least an image** or title to be valid.
```
Comment (cbikar) [inline pages/Documents/RichMessaging/ViberTemplates/limitations.md:15]: @joshuajott should be expand the bold text to also include "or title" ?

File: pages/Documents/RichMessaging/ViberTemplates/quick-replies-template.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+  categoryname: "Rich Messaging"
+  documentname: Viber Templates
+  permalink: viber-templates-quick-replies-template.html
+  root-link: true
+  indicator: messaging
+---
+
+### Overview
+
+Quick Replies for Viber include the question/statement text and a set of up to 11 replies that each contain:
```
Comment (cbikar) [inline pages/Documents/RichMessaging/ViberTemplates/quick-replies-template.md:16]: @joshuajott can you mention that quick replies will be presented to the user as custom keyboard?

File: _data/capabilities.yaml (hunk 1 above)
Comment (joshuajott) [reply _data/capabilities.yaml]: reverted.

**Review by cbikar — APPROVED** [review]

**Pull request merged**